Material.Demo/MainWindow.xaml.cs

```diff
@@ -21,20 +33,57 @@ public class MainWindow : Window {
         private void InitializeComponent() {
             AvaloniaXamlLoader.Load(this);
             _paletteHelper = new PaletteHelper();
-            this.FindControl<ToggleSwitch>("BaseThemeCheckBox").IsChecked = _paletteHelper.GetTheme().GetBaseTheme() == BaseThemeMode.Dark;
+
+            UseMaterialUIDarkTheme(); 
+
+            #region Control getter and event binding
+            NavDrawerSwitch = this.Get<ToggleButton>(nameof(NavDrawerSwitch));
+
+            DrawerList = this.Get<ListBox>(nameof(DrawerList));
+            DrawerList.PointerReleased += DrawerSelectionChanged;
+
+            PageCarousel = this.Get<Carousel>(nameof(PageCarousel));
+            #endregion
+
+            //this.FindControl<ToggleSwitch>("BaseThemeCheckBox").IsChecked = _paletteHelper.GetTheme().GetBaseTheme() == BaseThemeMode.Dark;
         }
 
         public void BaseThemeChanged(object sender, RoutedEventArgs args)
         {
             if (!(sender is ToggleSwitch checkBox)) return;
+            ChangeDarkMode(checkBox.IsChecked!.Value);
+        }
+        public void ChangeDarkMode(bool isEnabled)
+        {
             var theme = _paletteHelper.GetTheme();
-            var baseThemeMode = checkBox.IsChecked!.Value switch {
-                true  => BaseThemeMode.Dark,
+            var baseThemeMode = isEnabled switch
+            {
+                true => BaseThemeMode.Dark,
                 false => BaseThemeMode.Light
             };
             theme.SetBaseTheme(baseThemeMode.GetBaseTheme());
             _paletteHelper.SetTheme(theme);
         }
+
+        public void UseMaterialUIDarkTheme()
+        {
+            var theme = _paletteHelper.GetTheme();
+            theme.SetPrimaryColor(SwatchHelper.Lookup[MaterialColor.Blue200]);
+            theme.SetSecondaryColor(SwatchHelper.Lookup[MaterialColor.Pink200]);
+            theme.SetBaseTheme(BaseThemeMode.Dark.GetBaseTheme());
+            _paletteHelper.SetTheme(theme);
+        }
+        public void UseMaterialUILightTheme()
+        {
+            var theme = _paletteHelper.GetTheme();
+            theme.SetPrimaryColor(SwatchHelper.Lookup[MaterialColor.Blue]);
+            theme.SetSecondaryColor(SwatchHelper.Lookup[MaterialColor.Pink400]);
```

Comment:
Why do you toggling primary and secondary colors with dark/light theme?

Reply:
For temporarily testing case. Could be removed when implemented customize theme demo.